pkg/apiserver/logs/handler.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logs
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pingcap-incubator/tidb-dashboard/pkg/config"
+	"github.com/pingcap/kvproto/pkg/diagnosticspb"
+)
+
+const defaultSearchLogDuration = 24 * time.Hour
+
+// @Summary fetch logs
+// @Description fetch logs from TiDB, TiKV, PD
+// @Accept json
+// @Produce json
+// @Success 200 {string} string
+// @Router /logs [get]
+func (s *Service) fetchHandler(c *gin.Context) {
+	// TODO: using parameters provided by client
+	endTime := time.Now().UnixNano() / int64(time.Millisecond)
+	startTime := int64(0)
+	var searchLogReq = &diagnosticspb.SearchLogRequest{
+		StartTime: startTime,
+		EndTime:   endTime,
+		Levels:    nil,
+		Patterns:  nil,
+	}
+	var args = []*ReqInfo{
+		{
+			serverType: "tidb",
+			ip:         "127.0.0.1",
+			port:       "10080",
+			req:        searchLogReq,
+		},
+	}
+
+	taskGroupID := controller.AddTaskGroup(args)
+	err := controller.RunTaskGroup(taskGroupID)
+	if err != nil {
+		c.String(http.StatusInternalServerError, err.Error())
+		return
+	}
+	c.String(http.StatusOK, taskGroupID)
+}
+
+func (s *Service) listAllTasksHandler(c *gin.Context) {
```

Comment:
Please make sure to add necessary comments according to https://github.com/swaggo/swag so that all APIs are available in our Swagger UI (so that it will be also available in our JavaScript side).